source/docs/03-metadata.xml

```diff
@@ -1542,6 +1542,17 @@
                            <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample084.txt" parse="text"/></egXML>
                         </figure>
                      </p>
+                     <p><specList>
+                           <specDesc key="physMedium"/>
+                     </specList></p>
+                     <p><gi scheme="MEI">physMedium</gi> is intended to capture technical metadata in a narrative form. One should think of this element as a container for notes on or a synopsis of the physical characteristics of an item rather than a place to capture the characteristics themselves. It can used instead of or in addition to the structured sub-elements of <gi scheme="MEI">physDesc</gi>. For example,</p>
```

Comment:
@pe-ro Isn't this a bit far from the element definition, which is "Records the physical materials used in the source, such as ink and paper."?